### ISSUE TITLE
https://www.reddit.com/r/psychology/comments/4166g7/long_term_prescription_opioid_use_associated_with/

### ISSUE THREAD

**Issue by afandian** [CrossRef/event-data-enquiries#32]

**Comment by afandian** [comment]
Hi! This is the Event Data support bot! I noticed you linked to a webpage:

I found the following Event(s) for that URL:
 - [e37eaea3-003b-414e-822e-3fcbd61090a4](https://query.eventdata.crossref.org/events/e37eaea3-003b-414e-822e-3fcbd61090a4)

I found this by querying: http://query.eventdata.crossref.org/events?filter=source:reddit

Did I answer your support query? If so, feel free to close this ticket. If not, reply and we'll get back to you.

**Comment by crossref-support** [comment]
Hi! This is the Event Data support bot! I noticed you linked to a webpage:

I found the following Event(s) for that URL:
 - [e37eaea3-003b-414e-822e-3fcbd61090a4](https://query.eventdata.crossref.org/events/e37eaea3-003b-414e-822e-3fcbd61090a4)

I found this by querying: http://query.eventdata.crossref.org/events?filter=source:reddit

Did I answer your support query? If so, feel free to close this ticket. If not, reply and we'll get back to you.
